install.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
This script doesn't require bash (and bash may even not be installed in some distros), so "#!/bin/sh" would be better here.

Reply:
I pitch instead `#!/usr/bin/env sh`, which will get sh from the user's environment rather than hardcoding its location

Reply:
Sounds good, didn't know that was thing.

Reply:
> This script doesn't require bash (and bash may even not be installed in some distros), so "#!/bin/sh" would be better here.

Just checked, and this does use bash ([double square bracket conditionals being a bashism](http://mywiki.wooledge.org/BashFAQ/031)). If we wanted this to work with pure sh, these conditionals would need to be written using only the posix standard features. Alternatively, using bash here may just be the better idea

```suggestion
#!/usr/bin/env bash
```